src/js/typescript.js

```diff
@@ -49,9 +49,15 @@ export default class TypeScriptCompiler extends SimpleCompilerBase {
 
     d(output.diagnostics);
 
+    let sourceMaps;
+    if (output.sourceMapText) {
+      sourceMaps = JSON.parse(output.sourceMapText);
```

Comment:
This is for consistent return output between babel / TS, babel returns sourcemap as POJO and TS returns it as string, which has same structual interface to babel does. So either case is actually fine, between JSON.parse(ts) vs. JSON.stringify(babel). Would you prefer latter?